src/js/providers/html5.js

```diff
@@ -339,18 +339,14 @@ function VideoProvider(_playerId, _playerConfig) {
     }
 
     function _setVideotagSource(source) {
-        _androidHls = isAndroidHls(source);
         _audioTracks = null;
         _currentAudioTrackIndex = -1;
         if (!visualQuality.reason) {
             visualQuality.reason = 'initial choice';
             visualQuality.level = {};
         }
         _canSeek = false;
-        if (_androidHls) {
-            // Playback rate is broken on Android HLS
-            _this.supportsPlaybackRate = false;
-        }
+
```

Comment:
Let's get rid of this extra line of white space

Reply:
Removed